src/modules/users/UsersPage.cpp

```diff
@@ -403,23 +417,25 @@ UsersPage::onPasswordTextChanged( const QString& )
     }
     else
     {
-        bool ok = true;
         for ( auto pc : m_passwordChecks )
         {
             QString s = pc.filter( pw1 );
+
+            if ( ! ui->checkBoxValidatePassword->isChecked() &&
```

Comment:
I'd suggest factoring this out into a `isCheckEnabled( pc )` method of UsersPage so the special cases can be handled in one place.